doc/nrf/releases/release-notes-2.4.3.rst

```diff
@@ -0,0 +1,93 @@
+.. _ncs_release_notes_243:
+
+|NCS| v2.4.3 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, nRF70, and nRF91 Series.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+This patch release adds changes on top of the :ref:`nRF Connect SDK v2.4.0 <ncs_release_notes_240>`, :ref:`nRF Connect SDK v2.4.1 <ncs_release_notes_241>`, and :ref:`nRF Connect SDK v2.4.2 <ncs_release_notes_242>`.
+[TODO] The changes have an effect on Bluetooth Mesh.
```

Comment:
```suggestion
The changes have an effect on Bluetooth Mesh.
```

Reply:
I just left that as a placeholder. Maybe we need to be a bit more specific, but I believe you would give your input in https://nordicsemi.atlassian.net/wiki/spaces/NCS/pages/452001806/NCS+2.4.3+Release+highlights and I will get the more precise highlights from there.